mainnet-canisters.json

```diff
@@ -80,11 +80,11 @@
     "sha256": "a170bfdce5d66e751a3cc03747cb0f06b450af500e75e15976ec08a3f5691f4c"
   },
   "sns_root": {
-    "rev": "db5901a6e90b718918978ae5167b9c98d5aa7ab6",
-    "sha256": "d93e219b1c7f50098da1e4b19dae3e803014f7450e4fb61de5a0dc46dd94d6e1"
+    "rev": "aa91ecacdf3824e193e21b70e0127e8d3edab51a",
+    "sha256": "431cb333feb3f762f742b0dea58745633a2a2ca41075e9933183d850b4ddb259"
   },
   "swap": {
-    "rev": "4bed17bfc82cddc5691743db6228992cdc2740f4",
-    "sha256": "c265efedfb9268de8fa7647c04268a89339e4d02d670d146eb4564fa4a694a7f"
+    "rev": "aa91ecacdf3824e193e21b70e0127e8d3edab51a",
+    "sha256": "8313ac22d2ef0a0c1290a85b47f235cfa24ca2c96d095b8dbed5502483b9cd18"
```

Comment:
Looks like this PR needs a rebase on master; these changes were already merged.

Reply:
Yes it will be merged with master eventually, just not on this branch (see base).